battleship.py

```diff
@@ -62,7 +62,14 @@ def mousePressed(data, event, board):
 Returns: 2D list of ints
 '''
 def emptyGrid(rows, cols):
-    return
+    grid=[]
+    for i in range(rows):
+        l1 = []

```

Comment:
Use relevant variable name instead of l1

Reply:
Modified Variable Name

Reply:
Modified Variable name cols to col